Src/Scripts/Install-IronPython.ps1

```diff
@@ -0,0 +1,104 @@
+
+.EXAMPLE
+
+    PS>Invoke-WebRequest -Uri https://github.com/IronLanguages/ironpython3/releases/download/v3.4.0-beta1/IronPython.3.4.0-beta1.zip -OutFile IronPython.3.4.0-beta1.zip
+    PS>Install-IronPython -Path ~/ipyenv/v3.4.0-beta1 -ZipFile IronPython.3.4.0-beta1.zip -Force
+
+    The official binaries are downloaded from GitHub to the current directory and then the installation proceeds using the downloaded zip file. IronPython is installed into ~/ipyenv/v3.4.0-beta1, overwriting any previous installation in that location.
+    This example assumes that the installation script is in a directory on the search path ($env:PATH).
+#>
+[CmdletBinding()]
+Param(
+    # Target directory to which IronPython is to be installed.
+    [Parameter(Position=0, Mandatory)]
+    [string] $Path,
+
+    # The path to the downloaded zip file with IronPython binaries. If empty, the script will try to grab the package directly produced by the local build.
+    [string] $ZipFile,
+
+    # If the target path exists, it will be wiped clean beforehand.
+    [switch] $Force
+)
+
+$ErrorActionPreference = "Stop"
+
+$defaultVersion = "3.4.0-beta1"
+
+if (-not $ZipFile) {
+    # If zipfile path not given, assume that the script is in the standard location within the source tree
+    # and locate the zip archive in the standard location of the package target.
+    $projectRoot = $PSScriptRoot | Split-Path | Split-Path
+    $ZipFile = Join-Path $projectRoot "Package/Release/Packages/IronPython-$defaultVersion/IronPython.$defaultVersion.zip"
+}
+
+if (Test-Path $Path) {
+    if ($Force) {
+        if ((Resolve-Path $Path).Count -gt 1) {
+            Write-Error "Overwriting of multiple destinations not allowed: $Path"
+        }
+        Remove-Item -Path $Path -Force -Recurse
+    } else {
+        Write-Error "Path already exists: $Path"
+    }
+}
+
+# Unzip archive and move files into place
+$unzipDir = Join-Path $Path "zip"
+
+Expand-Archive -Path $ZipFile -DestinationPath $unzipDir
+Move-Item -Path (Join-Path $unzipDir "Lib") -Destination $Path
+Move-Item -Path (Join-Path $unzipDir "net6.0/*") -Destination $Path -Exclude "*.xml","*.dll.config"
+Remove-Item -Path $unzipDir -Recurse
+
+# Remove garbage files
+Remove-Item -Path (Join-Path $Path "Lib/__pycache__") -Recurse -ErrorAction Ignore
+
+# Create a startup script
+$ipyPath = Join-Path $Path "ipy.ps1"
+Set-Content -Path $ipyPath -Value @'
+#!/usr/bin/env pwsh
+dotnet (Join-Path $PSScriptRoot ipy.dll) @args
+'@
+if ($IsMacOS -or $IsLinux) {
+    chmod +x $ipyPath
+}
+
+# Add items that are missing in the zip file, directly from the bin directory
+if ($projectRoot) {
+    $binPath = Join-Path $projectRoot "bin/Release/net6.0"
+    Copy-Item (Join-Path $binPath "ipy") $Path
+    if ($IsMacOS -or $IsLinux) {
+        Copy-Item (Join-Path $binPath "Newtonsoft.Json.dll") $Path
```

Comment:
Are you using `Newtonsoft.Json` for something?

Reply:
No, but I wasn't sure if it wasn't being dynamically loaded by `ipy`, like `Mono.Unix.dll`. If you know for sure it's not needed, I'd rather not include it. But if it is not used, why is it being put in `bin` for normal builds? Used in testing?

Reply:
Looks like a transitive dependency for `Microsoft.NET.Test.Sdk` so not needed. I wonder if there's a way to not have it end up in the `bin`...